bin/s2nc.c

```diff
@@ -383,6 +404,9 @@ int main(int argc, char *const *argv)
             }
             psk_optarg_list[psk_list_len++] = optarg;
             break;
+        case 'E':
+            early_data = load_file_to_cstring(optarg);
```

Comment:
If `load_file_to_cstring` fails, is it guaranteed that `early_data` will be NULL? Should we exit with an error `if early_data == NULL` after this call?